Integrations/integration-Carbon_Black_Enterprise_Live_Response.yml

```diff
@@ -2097,7 +2101,8 @@ script:
         a new session. The session will be created and closed automatically.
     - name: path
       required: true
-      description: the path to save the resulting memory dump (on the endpoint).
+      description: 'The full path, including file name, to save the resulting memory dump on the endpoint
```

Comment:
since we deprecated this command let's write what replaces it

Reply:
done